spec/teacher_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../classes/teacher'
+
+describe Teacher do
+  before :each do
+    @teacher = Teacher.new('math', 19, 'sam')
+  end
+
+  it 'it should return true' do
+    expect(@teacher.can_use_services?).to be true
+  end
+
+  it 'it should return parameters' do
+    expect(@teacher.specialization).to eql('math')
+    expect(@teacher.age).to be(19)
+    expect(@teacher.name).to eql('sam')
+  end
+
+  it 'it should return instance' do
+    expect(@teacher).to be_instance_of Teacher
+  end
+end
```

Comment:
- [ ] [REQUIRED] Kindly fix the `teacher` class as it seems you are not providing what is expected. Expecting `19` but getting `sam`.

![Screenshot from 2022-04-20 18-46-55](https://user-images.githubusercontent.com/58692590/164291939-83b42576-6872-49aa-a136-987c4ec07928.png)